File: Submissions/T15/M1.py
nilai = list(map(int, input("Masukkan 5 nilai : ").split()))

rata = sum(nilai) / 5

if rata >= 80:
    grade = "A"
elif rata >= 70:
    grade = "B"
elif rata >= 60:
    grade = "C"
elif rata >= 50:
    grade = "D"
else:
    grade = "E"

print(f"Rata-rata : {rata:.2f}")
print(f"Grade : {grade}")

File: Submissions/T15/M1.cpp
#include <stdio.h>

int main() {
    int nilai[5];
    int total = 0;
    float rata;
    char grade;

    printf("Masukkan 5 nilai : ");

    for(int i = 0; i < 5; i++) {
        scanf("%d", &nilai[i]);
        total += nilai[i];
    }

    rata = total / 5.0;

    if(rata >= 80)
        grade = 'A';
    else if(rata >= 70)
        grade = 'B';
    else if(rata >= 60)
        grade = 'C';
    else if(rata >= 50)
        grade = 'D';
    else
        grade = 'E';

    printf("Rata-rata : %.2f\n", rata);
    printf("Grade : %c\n", grade);

    return 0;
}
